tasks.py

```diff
@@ -6,7 +6,6 @@
 import sys
 from enum import Enum
 from typing import Optional
-
```

Comment:
Let's not remove this space because normally, there is space after imports in Python code.